Library/Homebrew/download_queue.rb

```diff
@@ -294,7 +294,21 @@ def message_with_progress(downloadable, future, message)
 
       max_phase_length = 11
       phase = format("%-<phase>#{max_phase_length}s", phase: downloadable.phase.to_s.capitalize)
-      progress = " [#{phase} #{formatted_fetched_size}/#{formatted_total_size}]"
+      progress_bar = if future.fulfilled? || downloadable.phase != :downloading
+        ""
+      else
+        percent = if (total_size = downloadable.total_size)
+          (fetched_size.to_f / [1, total_size].max).clamp(0.0, 1.0)
+        else
+          0.0
+        end
+        bar_length = [4, available_width / 5].max
+        bar_used = (percent * bar_length).round
+        bar_completed = "━" * bar_used
+        bar_pending = "┈" * (bar_length - bar_used)
+        "#{bar_completed}#{bar_pending} "
```

Comment:
```suggestion
        bar_completed = "=" * bar_used
        bar_pending = " " * (bar_length - bar_used)
        "[#{bar_completed}#{bar_pending}] "
```
and perhaps adding a `=>` somewhere makes it more closely resemble `curl --progress-bar` which I think looks a little nicer. Could you rework to take that approach?